packages/web-components/fast-foundation/CEMToMarkdown.mjs

```diff
@@ -216,7 +216,24 @@ function replaceJSDOCLinksWithMDLinks(text)
         {
             text = text.replace(jsdocLink, "[" + linkParts[1].trim() + "](" + linkParts[0].trim() + ")");
         }
-        startIndex = text.indexOf('{@link',endIndex);
+        startIndex = text.indexOf('{@link');
+    }
+    return text;
+}
+
+function removeCommentBlocks(text)
```

Comment:
A regular expression that targets comment blocks would be safer than a while loop

Reply:
Done.

Reply:
I also replaced the JSDoc link replacement while loop with regular expressions.